### DISCUSSION TITLE
How add user id to logs?

### DISCUSSION THREAD

**Discussion by martinjanda** [pocketbase/pocketbase#5358] [Q&A] [answered]
In logs, I see data.auth: 'authRecord' atribute. I would like see data.user: 'user id'. 

Is it possible to achieve this from Go with some configuration or do I have to make a hook for each operation and duplicate each request logging?

If I have multiple hooks registered for one operation type and collection, are they executed in the order they appear in the Go code?

**Comment by ganigeorgiev** [comment, marked as answer]
It is not possible (_this is done primarily because of security and GDPR concerns since otherwise the logs no longer will be anonymous; this behavior may change in the future as there were several requests about this in the past but for now it is left for later consideration_).

> If I have multiple hooks registered for one operation type and collection...

Yes, multiple handlers attached to a single hook fire in the order they were registered. If you are not experiencing that, please provide a code sample illustrating the problem.

**Comment by martinjanda** [comment]
Thank you. 

I don't have a problem. I was just wondering if multiple hooks are done sequentially and in registered order or run in parallel.
